Tests/PackageLoading/ConventionTests.swift

```diff
@@ -45,6 +47,30 @@ class ConventionTests: XCTestCase {
                 moduleResult.checkSources(root: "/", paths: "Foo.swift")
             }
         }
+
+        // Single swift module inside Sources.
+        fs = InMemoryFileSystem()
+        try fs.createEmptyFiles("/Sources/Foo.swift",
+                                "/Sources/Bar.swift")
+
+        PackageBuilderTester(name, in: fs) { result in
+            result.checkModule(name) { moduleResult in
+                moduleResult.check(c99name: name, type: .library, isTest: false)
+                moduleResult.checkSources(root: "/Sources", paths: "Foo.swift", "Bar.swift")
+            }
+        }
```

Comment:
I think this would be a good place to also have "negative" tests, so for example check that `{/Sources/a.swift, root.swift}` is an error. Likewise for `{/Sources/a.swift,/Sources/subdir/b.swift}` (which I honestly can't remember what it parses as).